### PR TITLE
az-vm: add page

### PR DESCRIPTION
**Pull request by mfrw**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #6141

### CODE REVIEW

File: pages/common/az-vm.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# az vm
+
+> Manage virtual machines in Azure.
+> Part of azure-cli `az`.
```
Comment (CleanMachine1) [inline pages/common/az-vm.md:4]:
```suggestion
> Part of azure-cli's `az`.
```
```suggestion
> Part of azure-cli, `az`.
```

Comment (marchersimon) [reply pages/common/az-vm.md:4]: I prefer the first one

File: pages/common/az-vm.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+> More information: <https://docs.microsoft.com/cli/azure/vm>.
+
+- List details of availible Virtual Machines:
+
+`az vm list`
+
+- Create an `UbuntuServer 18.04 LTS` Virtual Machine and generate ssh keys:
+
+`az vm create --resource-group {{rg}} --name {{myVM}} --image {{Canonical:UbuntuServer:18.04-LTS:latest}} --admin-user {{azureuser}} --generate-ssh-keys`
```
Comment (marchersimon) [inline pages/common/az-vm.md:13]:
```suggestion
`az vm create --resource-group {{rg}} --name {{vm_name}} --image {{Canonical:UbuntuServer:18.04-LTS:latest}} --admin-user {{azureuser}} --generate-ssh-keys`
```

File: pages/common/az-vm.md (hunk 1 above)
Comment (mfrw) [reply pages/common/az-vm.md:4]: Thoughts about this:
```suggestion
> Part of `az`.
```

File: pages/common/az-vm.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Stop a Virtual Machine:
+
+`az vm stop --resource-group {{rg}} --name {{myVM}}`
```
Comment (mfrw) [inline pages/common/az-vm.md:17]:
```suggestion
`az vm stop --resource-group {{rg}} --name {{vm_name}}`
```

File: pages/common/az-vm.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Deallocate a Virtual Machine:
+
+`az vm deallocate --resource-group {{rg}} --name {{myVM}}`
```
Comment (mfrw) [inline pages/common/az-vm.md:21]:
```suggestion
`az vm deallocate --resource-group {{rg}} --name {{vm_name}}`
```

File: pages/common/az-vm.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+- Start a Virtual Machine:
+
+`az vm start --resource-group {{rg}} --name {{myVM}}`
```
Comment (mfrw) [inline pages/common/az-vm.md:25]:
```suggestion
`az vm start --resource-group {{rg}} --name {{vm_name}}`
```

File: pages/common/az-vm.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+- Restart a Virtual Machine:
+
+`az vm restart --resource-group {{rg}} --name {{myVM}}`
```
Comment (mfrw) [inline pages/common/az-vm.md:29]:
```suggestion
`az vm restart --resource-group {{rg}} --name {{vm_name}}`
```

File: pages/common/az-vm.md (hunk 1 above)
Comment (mfrw) [reply pages/common/az-vm.md:4]: > I prefer the first one

How about we just mention `az` ?

Comment (marchersimon) [reply pages/common/az-vm.md:4]: Even better.

**Review by marchersimon — APPROVED** [review]
Thanks @mfrw

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**